src/txkube/testing/strategies.py

```diff
@@ -252,6 +252,16 @@ def podspecs():
     """
     return builds(
         v1.PodSpec,
+        activeDeadlineSeconds=one_of(
+            none(),
+            integers(min_value=0, max_value=2 ** 63 - 1),
```

Comment:
Should this not be 2 ** 64 - 1?  Or is this a signed integer?  And if so, why is its minimum value 0?

I admit that having a negative deadline makes no sense, but then it also makes no sense to a use a signed integer.